addons/mail_group/views/mail_group_views.xml

```diff
@@ -0,0 +1,122 @@
+                            'invisible': [('privacy', '!=', 'groups')],
+                            'required': [('privacy', '=', 'groups')],
+                        }"/>
+                        <field name="moderation"/>
+                        <field name="is_moderator" invisible="1"/>
+                        <field name="can_manage_members" invisible="1"/>
+                    </group>
+                    <notebook>
+                        <page name="members" string="Members" attrs="{'invisible': [('can_manage_members', '=', False)]}">
+                            <field name="member_ids">
+                                <tree editable="top">
+                                    <field name="email" attrs="{'readonly': [('partner_id', '!=', False)]}" force_save="1"/>
+                                    <field name="partner_id"/>
+                                </tree>
+                            </field>
+                        </page>
+                        <page name="moderation" string="Moderation" attrs="{'invisible': [('moderation', '=', False)]}">
+                            <group>
+                                <field name="moderator_ids" widget="many2many_tags"/>
+                                <field name="moderation_notify"/>
+                                <field attrs="{'invisible': [('moderation_notify', '=', False)], 'required': [('moderation_notify', '=', True)]}" name="moderation_notify_msg"/>
+                                <field name="moderation_guidelines"/>
+                                <field attrs="{'required':[('moderation_guidelines', '=', True)]}" name="moderation_guidelines_msg"/>
+                            </group>
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id="mail_group_view_search" model="ir.ui.view">
+        <field name="name">mail.group.view.search</field>
+        <field name="model">mail.group</field>
+        <field name="arch" type="xml">
+            <search string="Search Mailing group">
+                <field name="name"/>
+                <field name="alias_name"/>
+                <separator/>
+                <filter string="Moderated" name="moderation" domain="[('moderation', '=', True)]"/>
+                <group expand="0" string="Group By">
+                    <filter string="Moderation" name="Moderation" context="{'group_by':'moderation'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+    <record id="mail_group_action" model="ir.actions.act_window">
+        <field name="name">Mailing group</field>
+        <field name="res_model">mail.group</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+    <menuitem id="mail_group.mail_group_menu" action="mail_group.mail_group_action" name="Mail Group" parent="mail.mail_menu_technical" sequence="50"/>
```

Comment:
I suggest to have a mail_group_menus.xml file holding all menus (easier to maintain / reorder menus).